modules/Microsoft.Web/staticSites/config/deploy.bicep

```diff
@@ -0,0 +1,33 @@
+@allowed([
+  'appsettings'
+  'functionappsettings'
+])
+@description('Required. Type of settings to apply.')
+param kind string
+
+@description('Required. App settings.')
```

Comment:
Would it make sense to pass properties as separated parameters for better parameter validation?

Reply:
I am not sure how it would improve parameter validation. The object passed for the settings themselves has to be a object in both cases, that currently cannot be validated by bicep anyway. Additionally for a deployment within the StaticSites resource it is abstracted into two separate deployments anyway, see 144, 153 in the main deploy of StaticSites.

Reply:
Agreed. It's not like there is any documentation anyways :)